e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -98,6 +98,7 @@ test.describe('Example Imagery', () => {
         await page.mouse.move(imageCenterX, imageCenterY);
 
         // Pan Imagery Hints
+        console.log(process.platform);
```

Comment:
Remove please :)

Reply:
@shefalijoshi we need a way to monitor for what's being reported from the system headlessly, remotely and irreproducible...ly?

I think we should keep this in place so that we can understand what the platform was when the test was executed